mappings/UofM.py

```diff
@@ -75,6 +70,40 @@ def formatIdentifiers(self):
                     return newISBNList
                 
         return self.record.identifiers
+    
+    def formatSubjects(self):
+        subjectList = []
+
+        if '|' in self.record.subjects:
+            subjectList = self.record.subjects.split('|')
+            newSubjectList = [f'{subject}||' for subject in subjectList] 
+            return newSubjectList
+        else:
+            subjectList.append(f'{self.record.subjects}||')
+            return subjectList
+    
+    def formatRights(self):
+        # Parse rights codes
```

Comment:
I think the code is clear that we are parsing the rights codes. My rec is to remove it!

Reply:
I agree